### PR TITLE
Add advice section to Readme

### PR DESCRIPTION
**Pull request by subshad**
Given that this is common advice to speakers (especially for this format at the current venue), add some things to think about for a prospective speaker.

### CODE REVIEW

**Comment by nkbt** [comment]
I have some:

1. http://www.slideshare.net/LookAtMySlides/codeware <-- pretty much must read 😸 
2. Considering fairly small screens, use very large fonts, minimum code listings and bullet-points. They are unreadable unless you are sitting in the first or second row
3. For the same reason live demos do not work out that well, I'd opt in for gifs instead (zoomed-in, full-slide). You can use LICECap - http://www.cockos.com/licecap it is awesome

File: README.md [hunk 1]
```diff
@@ -26,6 +26,14 @@ Make sure you fill out the following information:
 
 [**Submit your talk now**](https://github.com/reactsydney/talks/issues/new?body=**Talk%20Date**%0A%0A%0A**Talk%20Title**%0A%0A%0A**Length**%0A%0A%0A**Twitter-sized%20Overview%20(140%20characters)**%0A%0A%0A**Detailed%20Overview**%0A)
 
+## Things to think about
+
+Our current venue doesn't use a projector, but instead uses 2 x flatscreen TVs. If you have code examples, we suggest you make them take up the entire screen with as large a font as possible (we'd like the folks at the back to see the code as well).
+
+We use a hand-held mike rather than a lapel or headset, so be comfortable presenting with one hand holding a mike.
+
+Think about where you need your laptop positioned for advancing slides based on where you like to stand while speakking (at the front of the room, or off to the side).
```
Comment (nkbt) [inline README.md]: * speaking

Comment (subshad) [inline README.md]: whoops, thanks :smile:

**Comment by subshad** [comment]
@nkbt Added your suggestions. What do you think?

**Comment by nkbt** [comment]
Looks good to me, thanks, @subshad!

**Comment by jesstelford** [comment]
Love it! Thanks @subshad :D

**Pull request merged**